splinterd/src/config/error.rs

```diff
@@ -12,6 +12,8 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+#[cfg(feature = "log-config")]
+use log4rs::config::runtime::ConfigErrors;
```

Comment:
Is this change still relevant?

Reply:
Doesn't look like it is, Ill take it out